lib/raidenclient/errors.ts

```diff
@@ -35,6 +35,10 @@ const errors = {
     message: 'unexpected error during raiden request',
     code: errorCodes.UNEXPECTED,
   },
+  MISSING_SECRET: {
+    message: 'providing only the secret hash is currently not supported - please provide a secret',
+    code: errorCodes.UNEXPECTED,
+  },
```

Comment:
You can remove it.

When running raiden, you can use  the exteranl-swap-resolver  from https://github.com/offerm/raiden.git.